examples/AdvancedOperations/AddDisplayUploadAd.cs

```diff
@@ -0,0 +1,198 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
+using System.Linq;
+using Google.Ads.GoogleAds.Lib;
+using Google.Ads.GoogleAds.Util;
+using Google.Ads.GoogleAds.V3.Common;
+using Google.Ads.GoogleAds.V3.Enums;
+using Google.Ads.GoogleAds.V3.Errors;
+using Google.Ads.GoogleAds.V3.Resources;
+using Google.Ads.GoogleAds.V3.Services;
+using Google.Protobuf;
+using static Google.Ads.GoogleAds.V3.Enums.DisplayUploadProductTypeEnum.Types;
+
+namespace Google.Ads.GoogleAds.Examples.V3
+{
+    /// <summary>
+    /// This code example adds a display upload ad to a given ad group. To get ad groups,
+    /// run GetAdGroups.cs.
+    /// </summary>
+    public class AddDisplayUploadAd : ExampleBase
+    {
+        /// <summary>
+        /// Main method, to run this code example as a standalone application.
+        /// </summary>
+        /// <param name="args">The command line arguments.</param>
+        public static void Main(string[] args)
+        {
+            AddDisplayUploadAd codeExample = new AddDisplayUploadAd();
+
+            Console.WriteLine(codeExample.Description);
+
+            // The Google Ads customer ID for which the call is made.
+            long customerId = long.Parse("INSERT_CUSTOMER_ID_HERE");
+
+            // The ID of the ad group to which the new ad will be added.
+            long adGroupId = long.Parse("INSERT_ADGROUP_ID_HERE");
```

Comment:
Do we usually do ADGROUP or AD_GROUP? Can you please double-check that and change it if necessary?

Reply:
You're correct, typo on my part. Fixed.